packages/@docs/demos/src/demos/core/Menu/Menu.demo.navigation.tsx

```diff
@@ -4,17 +4,25 @@ import { MantineDemo } from '@mantinex/demo';
 import { DemoMenuItems } from './_menu-items';
 
 const code = `
-import { Menu, Group } from '@mantine/core';
+import { Group, Menu } from '@mantine/core';
 
 function Demo() {
   return (
     <Group>
-      <Menu trigger="click-hover" loop={false} withinPortal={false} trapFocus={false} menuItemTabIndex={0}>
-        {/* ... menu items */}
-      </Menu>
-      <Menu trigger="click-hover" loop={false} withinPortal={false} trapFocus={false} menuItemTabIndex={0}>
-        {/* ... menu items */}
-      </Menu>
+      {Array(4)
+        .fill(0)
+        .map((e, i) => (
+          <Menu
+            key={i}
+            trigger="click-hover"
+            loop={false}
+            withinPortal={false}
+            trapFocus={false}
+            menuItemTabIndex={0}
+          >
+            <DemoMenuItems />
```

Comment:
This will throw an error if copy pasted into a project, replace `<DemoMenuItems />` with `{/* ... menu items */}`